spec/linter-stylelint-spec.js

```diff
@@ -263,4 +265,27 @@ describe('The stylelint provider for Linter', () => {
       expect(messages.length).toBe(0);
     });
   });
+
+  describe('will autofix what errors it can', () => {
+    it('saves fixed css if attemptFix is true', async () => {
+      atom.config.set('linter-stylelint.attemptFix', true);
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
+      const fixedEditor = await atom.workspace.open(doneFixCSS);
```

Comment:
So I'm having a strange problem when I attempt to use TextEditor.setText:
I've updated my main test thusly:
``` javascript
it('applies fixed css if attemptFix is true', async () => {
      const toFixCSS = `
.FixMe
{


  height: 0.5em;
    color: #EEE;
}
      `;
      const doneFixCSS = `
.FixMe
{
  height: 0.5em;
  color: #eee;
}
      `;

      atom.config.set('linter-stylelint.attemptFix', true);
      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
      const editor = atom.workspace.buildTextEditor(); // also trying atom.workspace.open()
      editor.setText(toFixCSS);
      await lint(editor);
      const result = editor.getText();
      expect(result).toBe(doneFixCSS);
    });
```
The tests blow up because of `path.dirname`:

``` bash
The stylelint provider for Linter
  will autofix what errors it can
    it applies fixed css if attemptFix is true
      Error: Deprecated function(s) path.dirname) were called.
        path.dirname is deprecated. Argument to `path.dirname` must be a string
        -----------------------------------------------------------------------
        path.dirname -- /Applications/Atom.app/Contents/Resources/app/src/electron-shims.js:9:10
        <unknown> -- /Users/bmcginnis/github/linter-stylelint/lib/index.js:114:27
        undefined.next -- null:null:null
        step -- /Users/bmcginnis/github/linter-stylelint/lib/index.js:7:273
        <unknown> -- /Users/bmcginnis/github/linter-stylelint/lib/index.js:7:443
        <unknown> -- /Users/bmcginnis/github/linter-stylelint/lib/index.js:7:99
```